src/components/VideoPlayerContexts/PlaybackContext.tsx

```diff
@@ -110,21 +97,17 @@ function PlaybackContextProvider({children}) {
         }),
         [updateCurrentlyPlayingURL, currentlyPlayingURL, originalParent, sharedElement, shareVideoPlayerElements, playVideo, pauseVideo, checkVideoPlaying],
     );
-    return <PlaybackContext.Provider value={contextValue}>{children}</PlaybackContext.Provider>;
+    return <Context.Provider value={contextValue}>{children}</Context.Provider>;
 }
 
 function usePlaybackContext() {
-    const context = useContext(PlaybackContext);
-    if (context === undefined) {
+    const playbackContext = useContext(Context);
+    if (!playbackContext) {
```

Comment:
Please make sure that there is no side effects due to updated condition, since before the check was for `undefined` only, now it will cover `null` value as well.

Reply:
Updating this conditions solves the possible problem of [trying to unpack null](https://github.com/Expensify/App/blob/bee752ac593f40d96391199f680b14466076ab9d/src/components/VideoPlayerContexts/VolumeContext.js#L9), which is present in all calls of these functions. It changes behaviour a bit but I think this is what we want.